server/handlers.go

```diff
@@ -501,8 +502,8 @@ func (s *Server) finalizeLogin(identity connector.Identity, authReq storage.Auth
 		email = email + " (unverified)"
 	}
 
-	s.logger.Infof("login successful: connector %q, username=%q, email=%q, groups=%q",
-		authReq.ConnectorID, claims.Username, email, claims.Groups)
+	s.logger.Infof("login successful: connector %q, username=%q, preferred_username=%q, email=%q, groups=%q",
+		authReq.ConnectorID, claims.Username, claims.PreferredUsername, claims.Email, claims.Groups)
```

Comment:
```suggestion
		authReq.ConnectorID, claims.Username, claims.PreferredUsername, email, claims.Groups)
```
Your latest force-push reverted this.

Reply:
Sorry, I haven't pulled before it! Thanks!